acceptance/analyzer_test.go

```diff
@@ -905,10 +905,6 @@ func assertAnalyzedMetadata(t *testing.T, path string) {
 
 func assertLogsAndRestoresAppMetadata(t *testing.T, dir, output string) {
 	layerFilenames := []string{
-		"launch-build-cache-layer.sha",
-		"launch-build-cache-layer.toml",
-		"launch-cache-layer.sha",
-		"launch-cache-layer.toml",
```

Comment:
The tests that call `assertLogsAndRestoresAppMetadata` don't have a cache created. When the app metadata has `cache=true`, we also check that the cache metadata has `cache=true`, and don't write the file if it is inconsistent.